db/migrate/20220601193903_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :author_id
```

Comment:
- [ ] This is supposed to be a foreign key which refer to table users, kindly double-check the way of writing proper references so that the foreign will stay named `author_id`